ddtrace/tracer.py

```diff
@@ -116,6 +118,7 @@ def __init__(
 
         self.enabled = asbool(get_env("trace", "enabled", default=True))
         self.context_provider = DefaultContextProvider()
+        self.propagator = HTTPPropagator()  # type: Propagator
```

Comment:
```suggestion
        self._propagator = HTTPPropagator()  # type: Propagator
```

Unless we have a reason for it to be public 🙂

Reply:
Also, `HTTPPropagator` is more of a namespace for a bunch of static methods, so probably no need to create instances.
```suggestion
        self._propagator = HTTPPropagator  # type: ClassVar
```